tests/test_pipelines.py

```diff
@@ -298,7 +298,7 @@ def test_score_sde_ve_pipeline(self):
 
         assert image.shape == (1, 256, 256, 3)
 
-        expected_slice = np.array([0.64363, 0.5868, 0.3031, 0.2284, 0.7409, 0.3216, 0.25643, 0.6557, 0.2633])
+        expected_slice = np.array([0.35396, 0.31874, 0.0, 0.16337, 0.18218, 0.14668, 0.34573, 0.01166, 0.18164])
```

Comment:
checking this at the moment with original code base at the moment

Reply:
Yeah this one I would be careful with, historically very weird and can vary across platform.

Reply:
i tested now on osx (cpu) and linux amd64 (cpu/gpu) let me run some more tests on another server

Reply:
yes, I believe this change doesn't look like it's due to different archs. etc. but rather some potential (hopefully) fix that caused this more significant change...

Reply:
There is something weird here - don't get the same results anymore with our VE and the original code. Sorry give me one more day, need to go a bit deeper into the original code

Reply:
sure yes I suspect there is something! Please take your time!

Reply:
Thanks!

Reply:
Ok double-checked it with the original repo (see https://github.com/huggingface/diffusers/pull/291). The new codebase got different results only because a different random function is used (`torch.randn(...)` instead of `torch.randnlike(...)`) => So the PR is all good for me :-)

Also ran this script a couple of times:
```python
#!/usr/bin/env python3
from diffusers import DiffusionPipeline
import torch


score_sde_sv = DiffusionPipeline.from_pretrained("google/ncsnpp-church-256")
score_sde_sv.to("cuda")

# Note this might take up to 3 minutes on a GPU
i = 33
torch.manual_seed(i)
image = score_sde_sv(num_inference_steps=1000)["sample"][0]

# save image
image.save(f"/home/patrick/images/ported_church_nsaapp_{i}_compare.png")
```

and results look good

Reply:
There are two things we could improve here:

1. Pass a generator through the pipeline is cleaner IMO (after having merged: https://github.com/huggingface/diffusers/pull/289)
2. This test is by far the longest test of the slow tests (takes 30+ seconds on GPU). Can we remove the number of steps from 300 to just 50 ? The model is also not that important after all

Reply:
sure i can do that